pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -301,6 +302,7 @@ public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
     private String collectionDate;
     @ColumnName(DBConstants.SEQUENCING_RESTRICTION)
     private String sequencingRestriction;
+    private String receivedBy;
```

Comment:
If we are adding it to the participant list, I think it should also get added to the mapping and have a `@ColumnName` on top of the field name here

Reply:
I wasn't planning on adding it to the index yet. I only wanted it on the sample pages for now

Reply:
yeah, let's write it in and see if we need it…